CMakeLists.txt

```diff
@@ -582,7 +582,7 @@ if (BUILD_TESTING)
                 add_test(NAME ${test_target}
                         COMMAND
                         pytest
-                        -x -n=${N} --maxfail=1 --reruns=0 --cache-clear -rpfsq
+                        -x -n=${N} --reruns=2 --durations=10 --cache-clear -rpfsq
```

Comment:
What is `-n` set to in nix? Does reducing N help the flakiness at all? I think we limit N outside of nix for this reason:
https://github.com/aws/s2n-tls/blob/eb40a123ae307333d5842d64a949c3b3eded2f98/tests/integrationv2/tox.ini#L21

Reply:
Nix is using auto as opposed to 2, correct. Using retries instead of limiting workers allows us to keep the runtimes reasonable, while we try to understand the root cause. Another caveat, CodeBuild is using graviton2 so the arm tests run a slower- we need the parallelism to compensate.

Reply:
Is there already an issue open to look into the root cause?

Reply:
Opened a new issue, specific to this PR, at https://github.com/aws/s2n-tls/issues/4568